core/tauri/src/app.rs

```diff
@@ -1087,6 +1104,38 @@ impl<R: Runtime> Builder<R> {
     self
   }
 
+  /// Sets the current platform's target name for the updater.
+  ///
+  /// By default Tauri looks for one of `linux`, `win32`, `win64`, `darwin-silicon` or `darwin-intel`
```

Comment:
I think this is not up to date anymore correct?

Reply:
oops didn't see this one, thanks